File: abc100/c/main.py
n = int(input())
an = input()

an = an.split()
an = [int(s) for s in an]
print(n)
print(an)

#n = 10
#an = [2184 2126 1721 1800 1024 2528 3360 1945 1280 1776]
count = 0
while True:
    flag = 0
    sum = 0
    for i in range(1,n):
        if(flag == True):
            break

        if (an[i] % 2) == 0:
            an[i] = an[i] // 2
            flag = True
        else:
            sum = sum + 1

    if(sum == n):
        break

    count = count + 1

print(count)

File: abc100/c/main.cpp
#include <bits/stdc++.h>
using namespace std;
#define rep(i, n) for (int i = 0; i < n; i++)
#define REP(i, x, n) for (int i = x; i < n; i++)
#define mxin(a, b) (a = max(a, b))
#define mnin(a, b) (a = min(a, b))
using vi = vector<int>;
using vvi = vector<vi>;
using ll = long long;
using vl = vector<ll>;
using vll= vector<vl>;
using pii = pair<int, int>;
using pll = pair<ll, ll>;
using si = set<int>;
using sl = set<ll>;

int main()
{
	int n;
    cin >> n;
    vi input(n);
    int count=0;

    rep(i,n) cin >> input[i];

    rep(i,n){
        while(input[i] % 2 == 0){
            input[i] = input[i] / 2;
            count++;
        }
    }

    cout << count << endl;
    return 0;
}
